packages/helm-charts/blockscout/values.yaml

```diff
@@ -7,8 +7,7 @@ promtosd:
 blockscout:
   image:
     repository: gcr.io/celo-testnet/blockscout
-    webTag: web
-    indexerTag: indexer
+    tag: latest
```

Comment:
Is there a reason you used `latest` specifically here? We generally use specific tags because docker will often not pull if it already has a tag

Reply:
Hi @nambrot, 
No any specific reason. Now there is not need for two different images, so I picked `latest` as default value, but if we are versioning or any other patter I think it is very appropriate.